Makefile

```
@@ -52,6 +59,62 @@ $(services): target/spec $(openapi-generator-jar)
 	mv $(output)/$(models)/$@ $(models)/$@
 	mv $(output)/$(models)/JSON.java $(models)/$@
 
+# Full service + models automation
+bigServices:=balanceplatform checkout storedValue payments payout management legalentitymanagement transfers
+singleFileServices:=balancecontrol binlookup dataprotection storedvalue posterminalmanagement recurring
+
+all: $(bigServices) $(singleFileServices)
```

The `all` target is invoked when you run `make` (without any target). I would rather to have two targets, `models` for when generating only models and `services` for both models and service classes.

```suggestion
services: $(bigServices) $(singleFileServices)
```